relay-dynamic-config/src/defaults.rs

```diff
@@ -142,17 +146,18 @@ fn span_metrics() -> impl IntoIterator<Item = MetricSpec> {
             category: DataCategory::Span,
             mri: "d:spans/exclusive_time@millisecond".into(),
             field: Some("span.exclusive_time".into()),
-            condition: Some(
-                (is_db.clone() | is_resource.clone() | is_mobile.clone() | is_http.clone())
-                    & duration_condition.clone(),
-            ),
+            condition: None,
```

Comment:
Should we still apply `duration_condition` here? Asking because even `span_per_op` and `spans_per_segment` apply the duration condition.

Reply:
I don't think we'd want to apply the duration condition here. We want to track absolutely all spans here and the condition was mainly created for a mobile specific use case.